pages/Configuring/Variables.md

```diff
@@ -356,8 +356,11 @@ _Subcategory `group:groupbar:`_
 | enabled | enables groupbars | bool | true |
 | font_family | font used to display groupbar titles, use `misc:font_family` if not specified | string | [\[Empty]] |
 | font_size | font size of groupbar title | int | 8 |
+| font_weight_active | font weight of active groupbar title | string | normal |
+| font_weight_inactive | font weight of inactive groupbar title | string | normal |
```

Comment:
misses some explanations. Maybe add a new type `font_weight` that you describe above?